pycoshark/mongomodels.py

```diff
@@ -102,8 +102,8 @@ class TravisBuild(Document):
     def __repr__(self):
         return "<TravisBuild vcs_system_id:%s commit_id:%s number:%s duration:%s event_type:%s " \
                "pr_number:%s started_at:%s finished_at:%s stages:%s>" % \
-               (self.vcs_system_id, self.commit_id, self.number, self.duration, self.event_type, self.pr_number,
```

Comment:
Mixed functional and style changes, please create a separate PR for style changes in the future.

Reply:
I didn't intend to do this. It's my stupid IDE. I will revert it